docs/index.md

```diff
@@ -47,48 +49,92 @@ Older plugin that uses less resources than GLideN64, but has visual glitches (do
 
 Simple audio plugin that disables sound entirely. Does not seem to cause desyncs.
 
+## Configure your Controller
 
-### Controller
+=== "N64 / Hori (N64 compatible controllers)"
 
-#### raphnetraw 1 player NET version `Recommended`{ .badge data-md-color-primary="green" } `Original`{ .badge data-md-color-primary="cyan" } `Hori`{ .badge data-md-color-primary="cyan" } { #raphnet1net data-toc-label="raphnetraw 1 player NET" }
+    If your controller can plug into an N64, you'll want a [Raphnet N64 to USB adapter](https://www.raphnet-tech.com/products/n64_usb_adapter_gen3/index.php). Select the `raphnetraw 1 player NET version` controller plugin. No configuration required!
+    
+    If you have any other N64 to USB adapter, you'll need to select an N-Rage input plugin and configure your bindings manually.
 
-Best plug and play adapter for original and hori controllers. Both single and dual port versions work. No configuration necessary. For offline multiplayer use `raphnetraw for Project64 version`.
+=== "Gamecube"
 
-#### N-Rage `Xbox`{ .badge data-md-color-primary="cyan" } `Playstation`{ .badge data-md-color-primary="cyan" } `Keyboard`{ .badge data-md-color-primary="cyan" } { #nrage data-toc-label="N-Rage" }
+    If you have a Gamecube to USB adapter, follow the instructions on the [Gamecube Controller Configuration](/gamecube) page.
+    
+    If you have a Gamecube to N64 adapter for console play, you can connect it to an N64 to USB adapter for maximum accuracy and follow the N64 Compatible Controllers instructions above.
 
-Used with 3rd-party controllers/adapters, like Xbox/PS3/PS4/Keyboard. There are 3 versions available, and up to preference. You must map your own buttons, and **enable RAW data**.
+=== "Keyboard"
 
+    To use keyboard, first choose *N-Rage's Direct-Input8 1.61* as your input plugin. Then go to Configure Controller Plugin:
+    
+    1. Check Raw Data. **If you skip this step, netplay will not work!**
+    
+    2. Press Assign Keys. In the top right, choose "Keyboard".
 
-### Cheats
+    3. Bind your keys according to preference and choose Apply.
 
-Make sure the `1. Netplay Required` cheat is checked when you set up Project64KSE for the first time. This is a one time process and **does not** need to be done every time you launch the emulator. Start the game first to apply the cheat. If you use cheats other than `1. Netplay Required` you may need to recheck your cheats every time you launch the game.
+=== "Xbox / Other USB"
 
+    If you're using Windows 7, select N-Rage as your controller plugin, then choose Configure Controller Plugin and set your bindings as you like. **Ensure RawData is checked** under controller pak settings 
+    
+    If you're using Windows 8 or newer, you may need to download a premade N-Rage config file. See the [Xbox Controller Configuration](/xbox) page for details.
 
-### Start Netplay
+---
+
+## Start Netplay
+!!! Danger inline end "Ethernet Highly Recommended"
```

Comment:
Using this to discuss multiple parts parts of this page. I think it might look better not having `inline end` here, but I'm open to keeping it if you feel strongly about it. This will appear across the end page on mobile, fwiw.

And speaking of inline, I wonder if inlining the images would look better for the `Choose Rom Directory` and `Azimer's` sections? Rom Directory might be too small to look good, but I think Azimer's might be fine? Reference on how to do it is [here](https://squidfunk.github.io/mkdocs-material/reference/images/#image-alignment).

Reply:
Removed the inline for the ethernet warning in https://github.com/smash64-dev/smash64-online/pull/10/commits/da19be5987fa6d2f4230dea0e60b1096e06ed60f. I thought it was cute but don't have a strong opinion. I tried aligning those other images just now, but can't get them to look right, too much of the rest of the text ends up flowing around them depending on the width of the window. Maybe there's a formatting trick that would help. I agree that Azimer's is roughly the right size and the text on the other one is too short.